### PR TITLE
368: Storefront UI bumped to 0.7.12

### LINKED ISSUE #368: Missing page number in pagination if total page number is even and by 1 greater than visible pages
## Current behavior

Missing page number in pagination if total page number is even and by 1 greater than visible pages.

## Expected behavior

First and last page numbers should be always displayed.

## Steps to reproduce the issue

1. Go to https://capybara.storefrontcloud.io/men/tops-men/tops-12?page=3
   Total page number is 4.
2. Click page number 2.
   Total page number is 3.

pagination

## Can you handle fixing this bug by yourself?

- [ ] YES
- [ ] NO

## Environment details

- Browser: any
- OS: any
- Code Version: latest

## Additional information
Issue probably related to https://github.com/DivanteLtd/storefront-ui/issues/1182

### PR DESCRIPTION
**Pull request by psmyrek**
### Related Issues

Closes #368

### Short Description and Why It's Useful

This PR updates Storefront UI to v0.7.12 that fixes pagination issue.
Some minor changes were needed to support v0.7.12 by Capybara theme.

### Screenshots of Visual Changes before/after (If There Are Any)

**IMPORTANT NOTICE** - Remember to update `CHANGELOG.md` with description of your change

### Contribution and Currently Important Rules Acceptance

- [x] I read and followed [contribution rules](https://github.com/DivanteLtd/vsf-capybara/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed / Improved
 
 - Adjusted the error page (#398)
+- Missing page number in pagination if total page number is even and by 1 greater than visible pages (#368)
```
Comment (gibkigonzo) [inline CHANGELOG.md:14]: where is that change? :) I don't see changes in pagination :thinking:

Comment (psmyrek) [reply CHANGELOG.md:14]: This PR https://github.com/DivanteLtd/storefront-ui/pull/1189 has been included in v0.7.12 which fixes pagination issue in Capybara.

**Review by Fifciu — CHANGES_REQUESTED** [review]
Please resolve conflicts

**Comment by dt2patel** [comment]
Issue has no longer recreated.

**Pull request closed without merging**